configs/project-refs.conf

```diff
@@ -1,5 +1,6 @@
 vars.uris: {
   acyclic-uri:                    "https://github.com/lihaoyi/acyclic.git#cfaa2465105871e41681985c325ce262d1a6460a"
+  airframe-url:                   "https://github.com/wvlet/airframe.git#46b58ef6d0322e4e02d3921b6ad43638f23cf0f9"
```

Comment:
Pointing to the latest master HEAD revision.

Reply:
s/url/uri/

I didn't spot the discrepancy, but Jenkins did: https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-integrate-community-build/3782/console

Reply:
ok. Fixed